src/test/scala/eventstore/SoftDeleteStreamITest.scala

```diff
@@ -65,6 +68,7 @@ class SoftDeleteStreamITest extends TestConnection {
     "allow setting json metadata on empty soft deleted stream and recreate stream not overriding metadata" in new SoftDeleteScope {
       deleteStream(hard = false)
       writeMetadata("""{"test":"test"}""")
+      expectNoMessage(100.millis) // Give ES a bit time
```

Comment:
could you explain this?

Reply:
Just me trying fix tests due to slow ES/CI. Same as above